pkg/globalnet/controllers/ipam/ipam.go

```diff
@@ -607,6 +696,26 @@ func (i *Controller) podUpdater(obj runtime.Object, key string) error {
 	return nil
 }
 
+func (i *Controller) svcExUpdater(obj runtime.Object, key string) error {
+	var svcEx mcsv1a1.ServiceExport
+
+	unstructObj := obj.(*unstructured.Unstructured).UnstructuredContent()
+	err := runtime.DefaultUnstructuredConverter.FromUnstructured(unstructObj, &svcEx)
+	if err != nil {
+		klog.Errorf("Failed to extract ServiceExport from unstructured Object %#v", unstructObj)
+	}
+
+	svc, err := i.serviceGetter(svcEx.Namespace, svcEx.Name)
+	if err != nil && !errors.IsNotFound(err) {
+		klog.Errorf("Failed to get service %s/%s", svcEx.Namespace, svcEx.Name)
```

Comment:
I think this should be an Infof

Reply:
Same as above. Error is something other than NotFound.